packages/devtools_app/lib/src/memory/memory_heap_tree_view.dart

```diff
@@ -1308,22 +1309,23 @@ class _ClassOrInstanceCountColumn extends ColumnData<Reference> {
     } else if (dataObject.isAnalysis && dataObject is AnalysisReference) {
```

Comment:
line 1308 is still displayed as an int not a String. Can you change the return type of the method to String?

Reply:
I've re-written to be clearer about count, display, getValue, etc.  Should be consistent.